dartdoc_test/example/lib/example.dart

```diff
@@ -52,53 +64,11 @@ int multiply(int x, int y) {
 /// ```
 void ignore() {}
 
-/// Should return: Expected to find ';'.
-/// ```dart
-/// final fact = factorial(5)
-/// print(fact); // 120
-/// ```
-int factorial(int n) {
-  if (n < 0) throw ArgumentError('Negative numbers are not allowed.');
-  return n == 0 ? 1 : n * factorial(n - 1);
-}
-
-/// Checks if a string is a palindrome.
-///
-/// This method ignores case and non-alphanumeric characters.
+/// You can ignore code sample by using `dartdoc_test:ignore_error` tag.
 ///
-/// Example:
-/// ```dart
-/// final isPalindrome = isPalindrome('A man, a plan, a canal, Panama');
-/// print(isPalindrome); // true
-/// ```
-bool isPalindrome(String s) {
-  var sanitized = s.replaceAll(RegExp(r'[^A-Za-z0-9]'), '').toLowerCase();
-  return sanitized == sanitized.split('').reversed.join('');
-}
-
-/// Should return: Local variable 'gcd' can't be referenced before it is declared.
 /// ```dart
-/// final gcd = gcd(48, 18);
-/// print(gcd); // 6
-/// ```
-int gcd(int a, int b) {
-  while (b != 0) {
-    var t = b;
-    b = a % b;
-    a = t;
-  }
-  return a;
-}
-
-/// Convert a string to a list of lines.
+/// // dartdoc_test:ignore_error
```

Comment:
if we can make this work, because they it won't show up in the generated dartdoc documentation.
```suggestion
// dartdoc_test:ignore_error
```

Reply:
Maybe, we could do this, but it not as pretty.
```
/// my summary
///
/// <!-- ignore:dartdoc_test -->
/// my example:
/// ```dart
/// ...
/// ```
```